packages/message-box-react/src/MessageBox.tsx

```diff
@@ -6,20 +6,60 @@ interface Props {
     children: ReactNode;
     fullWidth?: boolean;
     className?: string;
+    inverted?: boolean;
 }
 
 type messageTypes = "info" | "error" | "success" | "warning";
 
 function messageFactory(messageType: messageTypes) {
-    return function messageBox({ title, fullWidth, className = "", children }: Props) {
+    return function messageBox({ title, fullWidth, className = "", children, inverted }: Props) {
         const componentClassName = classNames("jkl-message-box", "jkl-message-box--" + messageType, className, {
             "jkl-message-box--full": fullWidth,
+            [`jkl-message-box--${messageType}--dark`]: inverted,
         });
 
+        const getIcon = (type: string) => {
```

Comment:
Tror det begynner å bli verdt å ta en diskusjon rundt håntering av ikoner. Vi skal jo ikke ha så mange av de, men syns vi begynner å strø de litt rundt omkring isteden for å samle de på et sted, hvor vi feks kan preloade de.

Reply:
Ja, det er sikkert verdt å ta opp igjen. Fordelen med sånn det er nå er jo at de ikke vil bli brukt utenfor kontekst, så får man vurdere om det veier opp for mindre effektivitet. Høres ut som et bra agendapunkt for et forum 😊

Reply:
Jag fråga och just dessa ikoner skulle brukar på något annat ställe, men det verkar ikke som det, därför jag la den lokalt.